WinImageBuilder.psm1

```diff
@@ -874,6 +874,42 @@ function Run-Sysprep {
     Remove-VM $Name -Confirm:$false -Force
 }
 
+function Get-ImageInformation {
+    Param(
+        [Parameter(Mandatory=$true)]
+        [string]$driveLetter
+    )
+
+    if (Test-Path "$driveLetter\Windows\system32\ntdll.dll") {
+        $versionString = (Get-Item "$driveLetter\Windows\system32\ntdll.dll").VersionInfo.ProductVersion
+        $OSVersion = $versionString.split('.')
+        $ImageVersion = @{
+            "Major" = $OSVersion[0];
+            "Minor" = $OSVersion[1];
+        }
+    } else {
+        Throw "Unable to determine OS Version"
+    }
+
+    if (Test-Path "$driveLetter\Windows\SysWOW64") {

```

Comment:
replace it with:

(Get-Item "$driveLetter\Windows\system32\ntdll.dll").Target -like "*amd64_microsoft-windows-ntdll*"